packages/esm-service-queues-app/src/index.ts

```diff
@@ -130,6 +130,14 @@ export const transitionPatientToLatestQueue = getAsyncLifecycle(
   },
 );
 
+export const transitionOverflowMenuItem = getAsyncLifecycle(
+  () => import('./transition-latest-queue-entry/transition-overflow-menu-item/transition-overflow-menu-item.component'),
+  {
+    featureName: 'transition patient to new queue to support overflow menu',
```

Comment:
Since the component `transitionPatientToLatestQueue` right above this has the feature name `transition patient to new queue`, this is a bit confusing. Maybe something like:
```suggestion
    featureName: 'overflow menu with action to transition patient to a new queue',
```